tests/integration/benchmark/llmperf/README.md

```diff
@@ -0,0 +1,23 @@
+There are 3 steps to use this benchmark tool:
+
+* Install llmperf
```

Comment:
could we add `pip install llmperf`, or link to https://github.com/ray-project/llmperf?tab=readme-ov-file#installation? some instructions on what is needed to install llmperf will be useful

Reply:
Can we also add a line about AWS creds? will this work for any aws account, or is expected to be run with a specific one?